src/js/stores/MarathonStore.js

```diff
@@ -217,10 +236,17 @@ var MarathonStore = Store.createStore({
       case ActionTypes.REQUEST_MARATHON_GROUPS_SUCCESS:
         MarathonStore.processMarathonGroups(action.data);
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_SUCCESS:
+        MarathonStore.processMarathonDeployments(action.data);
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ERROR:
         MarathonStore.processMarathonGroupsError();
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ERROR:
+        MarathonStore.processMarathonDeploymentsError();
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ONGOING:
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ONGOING:
```

Comment:
`MarathonStore.processOngoingRequest` is a no-op, so presumably both cases should be removed?